videoprocess/process.cfg.template

```diff
@@ -39,8 +39,8 @@ FILTER_TYPE: VAProcFilterNone
 #default value will be applied then.
 
 #5.1 Denoise filter paramters
- #(0.0 ~ 1.0, default 0.5)
-DENOISE_INTENSITY: 0.75
+ #(0.0 ~ 64, default 0)
+DENOISE_INTENSITY: 20
```

Comment:
What is reason to use 20?  Sometimes DN effect is not so obvious, what about using a more stronger intensity to see obvious DN effect easily?

Reply:
> What is reason to use 20? Sometimes DN effect is not so obvious, what about using a more stronger intensity to see obvious DN effect easily?

I picked one lucas test to check the result, that case DN should be 20. I can change to a larger one